src/routes/chat/components/message/components/ImageAttachment.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import s from './ImageAttachment.css';
+
+function renderImage(attachments) {
```

Comment:
This looks like a React component so name it with PascalCase, please

Reply:
you called the file `ImageAttachment.js`, so call the component `ImageAttachment`